src/components/NightModeOverlay/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import Footer from "components/Footer";
+import SignUpForm from "components/SignUpForm";
+
+import cx from 'classnames';
+import "./NightModeOverlay.scss";
+
+class NightModeOverlay extends PureComponent {
+
+  render() {
+
+    const {
+      className
+    } = this.props;
+
+      return (
+        <div
+          className={cx(
+            'NightModeOverlay absolute fixed vh100 vw100 flex flex-col z3 col-12 overflow-hidden',
+            className
+          )}>
+            <div className="flex flex-col md:flex-row col-12 justify-between">
+              <div className="NightModeOverlay__navigation flex flex-col col-1 p1">
+                <div className="small md:block none">Social Media:</div>
+                  <nav 
+                    aria-label="Navigation for Sanctuary Computer Social Media Links"
+                    className="NightModeOverlay__social-media-links md:block col-4 md:mb6 px1 pb1 small">
+                      <a href="www.sanctuary.computer" className="block">Twitter</a>
+                      <a href="www.sanctuary.computer" className="block">Instagram</a>
+                      <a href="www.sanctuary.computer" className="block">Medium</a>
+                      <a href="www.sanctuary.computer" className="none md:block">Github</a>
+                      <a href="www.sanctuary.computer" className="none md:block">Podcast</a>
```

Comment:
Let's add the real links

Reply:
this block is already built and the data comes from contentful

Reply:
I was planning on using `<List>` and social media data from Contentful in my next PR where I add a fetchContentfulData action to keep this PR smaller. :) @joshiefishbein should I add to this PR and include #41?

Reply:
Add it in if you feel like the implementation will be straightfoward. Separate it if you think it may require an extra unforeseen step.

Reply:
I'll save this for #41!